opensrp-app/src/main/java/org/smartregister/util/DatabaseMigrationUtils.java

```diff
@@ -136,4 +138,63 @@ public static void recreateSyncTableWithExistingColumnsOnly(SQLiteDatabase datab
         database.endTransaction();
 
     }
+
+
+    private static boolean tableExists(SQLiteDatabase database, String tableName) {
+        Cursor cursor = database.rawQuery("SELECT name FROM sqlite_master WHERE type='table' AND name= ?", new String[]{tableName});
+        boolean exists = false;
+        if (cursor != null) {
+            exists = cursor.getCount() > 0;
+            cursor.close();
+        }
+        return exists;
+    }
+
+    public static void createAddedECTables(SQLiteDatabase database, List<String> bindings, CommonFtsObject commonFtsObject) {
+        ArrayList<CommonRepositoryInformationHolder> bindTypes = org.smartregister.Context.bindtypes;
+        for (CommonRepositoryInformationHolder bindType : bindTypes) {
+            if (bindings.contains(bindType.getBindtypename()) && !tableExists(database, bindType.getBindtypename())) {
```

Comment:
The `List` contains method is a bit inefficient (especially for long lists), is it possible to replace `bindings` type to set?

Reply:
Take note that this list is defined in OpenSRP Context and changing to a set would break other classes or Apps.

The bindings  contains the tables that are managed by ec model. The list is there not large

Reply:
I meant changing it to a `Set` before passing it to this method?

Reply:
The tables may never be that many to observe any real benefit, but IMO it's just good practice.

Reply:
Changed to a `Set`

Reply:
Cool.